apps/console/src/features/feature-gate/controller/show-feature.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    Fragment,
+    FunctionComponent,
+    PropsWithChildren,
+    ReactElement,
+    createElement,
+    useContext
+} from "react";
+import { FeatureGateContext } from "../context/feature-gate";
+import { FeatureGateContextInterface, FeatureGateShowInterface } from "../model/feature-gate";
+
+/**
+ * Interface for show component.
+ */
+export interface AccessControlShowInterface {
+
+    /**
+     * Granular level resource permissions.
+     */
+    ifAllowed?: string;
```

Comment:
Shall we use a better name? For example, `componnetPath`/`componentId`?

Reply:
After offline discussion we will be sticking with this **ifAlllowed**